### PR TITLE
Import Alexandria

### PR DESCRIPTION
**Pull request by camjc**
- Add note to Readme about how to profile the webpack dist.
- Finally decide that react-bootstrap is a dev dependency and we
  should ship the parts we're exporting. This way we can load only the
  modules we use in Direct and Symphony etc.
- Update package.json to point to public npm and our git repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,3 +84,10 @@ Most of the time you will want to add:
 `--stateless`
 
 That makes a stateless component (pure function, no internal state).
+
+Build Profiling
+---------------
+
+To generate a profile: `npm run profile`
+
+Upload the new `stats.json` file to [Webpack Analyse Tool](http://webpack.github.io/analyse).
```
Comment (etaoins) [inline README.md]: Interesting they use the Queen's English in the URL. I thought you might've overcorrected but it checks out.

File: package.json [hunk 2]
```diff
@@ -21,13 +21,20 @@
     "release:minor": "npm version minor && npm publish && git push --follow-tags",
     "release:patch": "npm version patch && npm publish && git push --follow-tags"
   },
-  "repository": "",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Adslot/adslot-ui.git"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
```
Comment (etaoins) [inline package.json]: :+1:

**Comment by etaoins** [comment]
LGTM :green_heart:

File: README.md (hunk 1 above)
Comment (omgaz) [inline README.md]: :gb:

**Comment by omgaz** [comment]
Merging :ramen:

File: package.json (hunk 2 above)
Comment (runk) [inline package.json]: cc @bchew

**Pull request merged**